cogs/strike.py

```diff
@@ -1000,7 +1000,7 @@ async def _send_message_to_committee(
             name="discord",  # TODO: Make this user-configurable  # noqa: FIX002
         )
 
-        if not discord_channel:
+        if discord_channel is None:
```

Comment:
unnecessary change and consistency should be preferred where possible, recommend revert